src/client/components/HelpText/HelpText.css

```diff
@@ -0,0 +1,15 @@
+.helptext-component {
+    border: 3px;
+    float: left;
+    padding: 30px 40px;
+  }
+  .HelpText-message {
```

Comment:
In line one it is `helptext` and here is `HelpText`. Can you align both to one style?